dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/DataverseSession.java

```diff
@@ -112,6 +114,16 @@ public StaticPermissionQuery on(Dataverse d) {
         return permissionsService.userOn(user, d);
     }
 
+    public String getLocaleLanguageShortName() {
```

Comment:
1. This method is not necessary you can just use `getLocale().toLanguageTag()`.
2. If this method was necessary I would also go with pure function, which means that all required things are passed in the arguments, otherwise testing this method would be troublesome.